src/plans/D3MAavePlan.sol

```diff
@@ -19,41 +19,41 @@ pragma solidity ^0.8.14;
 import "./ID3MPlan.sol";
 
 interface TokenLike {
-    function totalSupply() external view returns (uint256);
+    function totalSupply()      external view returns (uint256);
```

Comment:
usually interfaces are not aligned, this would break previous patterns (see `dss-interfaces`)

Reply:
Ok, for TokenLike let's keep the alignment as before.